### ISSUE TITLE
Cursor doesn't change when hovering over new mood description

### ISSUE THREAD

**Issue by enoralecuyer** [ifmeorg/ifme#1730]
# Description

The cursor doesn't change when hovering over the description of a new mood, as we are about to type. To make this feature more accessible, I suggest to change the cursor to a type text.

This change is also needed for:
- new moment
- new category 
- new strategy
- new medication
- new group

# Screenshots
![newmoodcursor](https://user-images.githubusercontent.com/48727972/80432849-3b5d2780-88aa-11ea-80b8-71e64a64c33d.png)

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by julianguyen** [comment]
This is a really great find! Thanks for taking this on!!

**Comment by JayeshJanyani** [comment]
Hi @enoralecuyer, I am interested to take this up.

**Comment by enoralecuyer** [comment]
Hi @JayeshJanyani, thanks, you can assign it to yourself.

**Comment by klaffert** [comment]
@enoralecuyer @JayeshJanyani Hi, is this still an open issue? If so, I'd like to take this up. Thanks!

**Comment by enoralecuyer** [comment]
Hi @klaffert, this is still an open issue. @JayeshJanyani will be able to bring you more information as he took the issue a few days ago. Thanks for your interest!

**Comment by julianguyen** [comment]
Hey folks, please assign yourself to the issue if you're working on it :) Also working together could be a great way for multiple people to work on an issue!

**Comment by klaffert** [comment]
@enoralecuyer It seems like the type text cursor appears underneath the editing links on 'Description'. Here's what I'm seeing:

https://www.screencast.com/t/4qtGwdGf6

Do we want to make the whole 'Description' area to be a type text cursor? I'm thinking that since the editable links are within the text area, we'd want to keep it as is? 

Just want to ensure that I understand the bug fully! Thanks!

**Comment by enoralecuyer** [comment]
@klaffert Thank you for asking for clarification! 

I just logged in to the website and I confirmed that the cursor is now changing to a text cursor when hovering over the writing area, just as your video shows. The change has been made!

I agree with you to not make the whole area a type cursor, but simply the area where we write. Thanks!

**Comment by klaffert** [comment]
@enoralecuyer Great, thanks for the info! I'll close this issue for now. Let me know if there are questions.
